pynode/3.9/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-slim-buster
+FROM python:3.6-slim-buster
```

Comment:
Didn't you mean 3.9 ?

Also, don't forget dependabot
![image](https://user-images.githubusercontent.com/3366767/133620899-62b5b41e-a853-4aeb-89e9-3f5d7c88d8a7.png)